backend/app/views/spree/admin/customer_returns/_reimbursements_table.html.erb

```diff
@@ -20,8 +20,8 @@
         </td>
         <td><%= reimbursement.display_total %></td>
         <td>
-          <span class="state <%= reimbursement_status_color(reimbursement) %>">
-            <%= reimbursement.reimbursement_status %>
+          <span class="pill pill-<%= reimbursement.reimbursement_status %>">
+            <%= reimbursement.reimbursement_status.titleize %>
```

Comment:
Please do not transform strings like this, pass it through `I18n` or just leave it like it is.